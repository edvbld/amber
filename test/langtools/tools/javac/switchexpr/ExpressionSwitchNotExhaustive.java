/**
 * @test /nodynamiccopyright/
 * @compile/fail/ref=ExpressionSwitchNotExhaustive.out -XDrawDiagnostics ExpressionSwitchNotExhaustive.java
 */

public class ExpressionSwitchNotExhaustive {
    private String print(int i) {
        return switch (i) {
            case 42 -> "42";
            case 43 -> "43";
        };
    }
    private String e(E e) {
        return switch (e) {
            case A -> "42";
        };
    }
    enum E {
        A, B;
    }
}
