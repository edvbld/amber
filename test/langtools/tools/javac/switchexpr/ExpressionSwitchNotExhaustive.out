ExpressionSwitchNotExhaustive.java:8:16: compiler.err.not.exhaustive
ExpressionSwitchNotExhaustive.java:14:16: compiler.err.not.exhaustive
ExpressionSwitchNotExhaustive.java:27:23: compiler.err.var.might.not.have.been.initialized: s
- compiler.note.preview.filename: ExpressionSwitchNotExhaustive.java
- compiler.note.preview.recompile
3 errors
